apis/python/src/tiledbsoma/types.py

```diff
@@ -39,5 +40,7 @@
     pa.SparseCSCMatrix,
 ]
 
-PlatformConfig = Mapping[str, Any]
-"""The platform-configuration dictionary. May contain a ``tiledb`` key."""
+StorageType = Literal["array", "group"]
+"""How a SOMA object is stored."""
+TDBHandle = Union[tiledb.Array, tiledb.Group]
```

Comment:
given how the `tiledb` types are defined, this might be better as:
> TDBHandle = Union[tiledb.DenseArray, tiledb.SparseArray, tiledb.Group]

Reply:
Probably true. There is a lot of type-annotation–related stuff that I pretty much got to the point of "it works and I don't want to touch it" and I want to go back through and clean that up after this implementation work settles down.